README.md

```diff
@@ -145,6 +145,14 @@ For each field (question) that needs to be indexed, there must be a `filter_` pr
 same as the filename, then the _overridden_ `id` should be used here. (Compare with the manifest,
 where in all cases the filename should be used to specify the question.)
 
+We are using Elasticsearch multifields to support service aggregations by specific attributes (eg lot and category).
+We use a `raw` subfield so that tokenization is not performed on the values to make aggregation possible with natural
```

Comment:
Can we use the word 'multi-field' rather than 'subfield', as that seems to be what the elasticsearch documentation uses, and I got a bit confused looking up the wrong thing :)